### PR TITLE
Scrolling Intervention post.

### PR DESCRIPTION
**Pull request by PaulKinlan**
FYI: @petele @jpmedley

### CODE REVIEW

File: src/content/en/updates/2017/01/scrolling-intervention.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: We know that scrolling responsiveness is critical to the user's engagement with a website on mobile, yet touch event listeners often cause serious scrolling performance problems.
+
+{# wf_updated_on: 2017-01-10 #}
+{# wf_published_on: 2017-01-10 #}
+{# wf_tags: intervention,chrome56 #}
+{# wf_featured_image: /web/updates/images/generic/warning.png #}
+{# wf_featured_snippet: We know that scrolling responsiveness is critical to the user's engagement with a website on mobile, yet touch event listeners often cause serious scrolling performance problems. #}
```
Comment (petele) [inline src/content/en/updates/2017/01/scrolling-intervention.md]: Just a nit - but this is your hook, this is what shows up on the /web/updates/ landing page (and you can use HTML in here). You might want to make it a bit more exciting. For example:

> Scrolling responsiveness is critical to the user's engagement with a website on mobile, yet touch event listeners often cause serious scrolling performance problems! With `magic` you can avoid all that. Your users will love you and other developers will be envious of your **1337** skills.

File: src/content/en/updates/2017/01/scrolling-intervention.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+{% include "web/_shared/contributors/dtapuska.html" %}
+
+We know that scrolling responsiveness is critical to the user's engagement with
+a website on mobile, yet touch event listeners often cause serious scrolling
+performance problems. Chrome has been addressing this by allowing touch event
+listeners to be
+[passive](https://developers.google.com/web/updates/2016/06/passive-event-listeners?hl=en)
+(passing the `{passive: true}` option to `addEventListener()`) and shipping the
+[pointer
+events](https://developers.google.com/web/updates/2016/10/pointer-events) API.
```
Comment (petele) [inline src/content/en/updates/2017/01/scrolling-intervention.md]: Use absolute URLs without the fully qualified domain. Using the fully qualified domain breaks the .cn site.

File: src/content/en/updates/2017/01/scrolling-intervention.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+In the vast majority of cases, no breakage will be observed. But when breakage 
+does occur, the most common symptom is that scrolling happens when you don't 
+want it. In rare cases, developers may also notice unexpected **click** events 
+(when `preventDefault()` was missing from a `touchend` listener).
+
+In Chrome 56 and later, DevTools will log a warning when you call 
+`preventDefault()` in an event where the intervention is active.
+
+`touch-passive.html:19 Unable to preventDefault inside passive event listener due
```
Comment (petele) [inline src/content/en/updates/2017/01/scrolling-intervention.md]: I think backticks would work here, but you might do this indented 4 spaces instead

File: src/content/en/updates/2017/01/scrolling-intervention.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+In Chrome 56 and later, DevTools will log a warning when you call 
+`preventDefault()` in an event where the intervention is active.
+
+`touch-passive.html:19 Unable to preventDefault inside passive event listener due 
+to target being treated as passive. See 
+https://www.chromestatus.com/features/5093566007214080`
+
+Your application can determine whether it may be hitting this in the wild by 
+checking if calling preventDefault had any effect via the 
+[defaultPrevented](https://developer.mozilla.org/en-US/docs/Web/API/Event/defaultPrevented)
```
Comment (petele) [inline src/content/en/updates/2017/01/scrolling-intervention.md]: wrap defaultPrevented in backticks

File: src/content/en/updates/2017/01/scrolling-intervention.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+Your application can determine whether it may be hitting this in the wild by 
+checking if calling preventDefault had any effect via the 
+[defaultPrevented](https://developer.mozilla.org/en-US/docs/Web/API/Event/defaultPrevented) 
+property. 
+
+We've found that a large majority of impacted pages are fixed relatively easily
+by applying the
+[touch-action](https://developer.mozilla.org/en-US/docs/Web/CSS/touch-action)
+CSS property whenever possible. If you wish to prevent all browser scrolling and
+zooming within an element apply touch-action: none to it. If you have a
```
Comment (petele) [inline src/content/en/updates/2017/01/scrolling-intervention.md]: wrap touch-action: none in backticks

File: src/content/en/updates/2017/01/scrolling-intervention.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+only impact that most developers will notice as a result of this change. In some
+cases developers may notice unintended scrolling.
+
+Although it's still necessary to do so for mobile Safari, websites should not
+rely on calling `preventDefault()` inside of `touchstart` and `touchmove`
+listeners as this is no longer guaranteed to be honored in Chrome. Developers
+should apply the touch-action CSS property on elements where scrolling and
+zooming should be disabled to notify the browser before any touch events occur.
+To suppress the default behavior of a tap (such as the generation of a click
+event), call `preventDefault()` inside of a `touchend` listener.
```
Comment (petele) [inline src/content/en/updates/2017/01/scrolling-intervention.md]: Don't forget the comment block

File: src/content/en/updates/2017/01/scrolling-intervention.md (hunk 2 above)
Comment (PaulKinlan) [inline src/content/en/updates/2017/01/scrolling-intervention.md]: bah.

**Pull request merged**